src/ETISS.cpp

```diff
@@ -586,19 +589,36 @@ void etiss::Initializer::loadIniPlugins(std::shared_ptr<etiss::CPUCore> cpu)
         po_simpleIni->GetAllKeys(iter_section.pItem, keys);
         for (auto iter_key : keys)
         {
-            // get all values of a key with multiple values = value of option
-            CSimpleIniA::TNamesDepend values;
-            po_simpleIni->GetAllValues(iter_section.pItem, iter_key.pItem, values);
-            for (auto iter_value : values)
-            {
-                options[iter_key.pItem] = iter_value.pItem;
-                etiss::log(etiss::INFO,
-                           "    options[" + std::string(iter_key.pItem) + "] = " + std::string(iter_value.pItem));
-            }
+            //MINE 
+                // get all values of a key with multiple values = value of option
+                CSimpleIniA::TNamesDepend values;
+                po_simpleIni->GetAllValues(iter_section.pItem, iter_key.pItem, values);
+                try
+                {
+                    //function returns true false
+                    options[iter_key.pItem] = std::string(vm[std::string(iter_key.pItem)].as<std::string>());
+                    std::cout<<"\n"<< iter_section.pItem <<"::"<<iter_key.pItem<< " written from command line.\n";
+                    //std::cout<<std::string(iter_key.pItem)<<" set on command line to "<<options[iter_key.pItem];
+                    etiss::log(etiss::INFO,
+                                "    options[" + std::string(iter_key.pItem) + "] = " + std::string(vm[std::string(iter_key.pItem)].as<std::string>()));
+                }
+                catch(const std::exception& e)
```

Comment:
Exceptions should usually not be used for general control flow, but only in unexpected circumstances. Can you rewrite this to a check that looks for the value in the command line options first?